### ISSUE TITLE
Screen Reader Support Toggle shortcut does not work 

### ISSUE THREAD

**Issue by jooyoungseo** [rstudio/rstudio#6852]
### System details

    RStudio Edition : Desktop or Server
    RStudio Version : 1.3.957
    OS Version      : Windows
    R Version       : any

### Steps to reproduce the problem

1. Press Alt+Shift+/ (forward slash) to toggle "Screen Reader Support."
2. Tab to find "Yes" button and press enter in response to the following dialog popup:

> Are you sure you want to enable screen reader support? The application will reload to apply the change.

### Describe the problem in detail

Accessibility Support mode does not change between being enabled and disabled using shortcut key.

FYI, Help / Accessibility / Screen Reader Support menu does not make any change, either.

Currently, you can only toggle this mode via Help / Accessibility / Accessibility Options / Screen reader support checkbox.

### Describe the behavior you expected

This shortcut key must work reliably.

- [x] I have read the guide to submitting good bug reports at https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports . 
- [x] I have installed the latest version of RStudio and confirmed that the issue still persists.
- [x] If I am reporting a RStudio crash, I have included a diagnostics report. https://support.rstudio.com/hc/en-us/articles/200321257-Running-a-Diagnostics-Report
- [x] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

**Comment by jooyoungseo** [comment]
If you cannot reproduce this issue, please toggle on and off the following switch:

Help / Accessibility / Accessibility Options / Entire page has application role (requires restart)

I have found that "Screen Reader Support" toggle shortcut key (Alt+Shift+/) does not make any change after I played with the option above back and forth.

**Comment by jooyoungseo** [comment]
If this issue has to do with `role = "application"` switch in some ways, we may want to remove the "Entire page has application role" toggle from the accessibility options while setting application role as default.  My gut feeling is that the required restart from the application role has caused some breaks in regards to "Screen Reader Support" toggle switch.

**Comment by gtritchie** [comment]
On Safari / Mac I'm seeing a problem where hitting Enter always selects the "No" button even if I have tabbed to the Yes button. Have to use the spacebar to activate the "No" button.

The enter keys seems to work properly on Windows with Chrome or FireFox.

**Comment by jooyoungseo** [comment]
@gtritchie, Have you also tried with QuickNav *off*?

**Comment by gtritchie** [comment]
Yes, I have quicknav off. The problem happens even if I don't have a screen reader running. Again, this appears to be Mac-specific.

**Comment by jooyoungseo** [comment]
Aw, I see. That's so odd...

**Comment by gtritchie** [comment]
Suspect it's a general problem with the Yes / No dialog code that is used in a variety of places. No idea yet why it's Mac specific.

**Comment by gtritchie** [comment]
To capture some side conversation: traditionally in most dialog box keyboard models, there is a difference between hitting enter key and spacebar. Spacebar is for clicking the control that has keyboard focus, but enter is for activating the default button or action.

This model seems to be falling out of favor and used more and more inconsistently. It was really focused on traditional form-based dialogs, where enter meant you had filled in the form and were ready to submit it. When I try on Windows these days it seems that once I've navigated to a different button via keyboard, enter and spacebar do the same thing and click that button. No doubt both models can be found in active use, though.

**Comment by jooyoungseo** [comment]
Understood! That makes sense to me. :)

**Comment by gtritchie** [comment]
Also note that in 1.3-patch we have removed the "Entire page has application role" preference and checkbox so playing with that isn't important any more (for 1.3 best to test screen reader with it on, though).

**Comment by jooyoungseo** [comment]
I have confirmed this issue has to do with [#6928](https://github.com/rstudio/rstudio/issues/6928). I kept using "enter" key on yes button in the screen reader support toggle dialog box with JAWS on Windows; however, always "No" has been triggered.

**Comment by gtritchie** [comment]
This should now be fixed in latest 1.4 build via the fix for #6928.

**Comment by kfeinauer** [comment]
Verified fix in RSP 1.4.940-1, same as in #6928.
